<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceMonitorManagerWindow</class>
 <widget class="QWidget" name="ServiceMonitorManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>453</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="interfaceList">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
